bundles/org.openhab.core.config.discovery.addon.mdns/src/main/java/org/openhab/core/config/discovery/addon/mdns/MDNSAddonFinder.java

```diff
@@ -75,7 +75,7 @@ public MDNSAddonFinder(@Reference MDNSClient mdnsClient) {
     /**
      * Adds the given mDNS service to the set of discovered services.
      *
-     * @param device the mDNS service to be added.
+     * @param service the mDNS service to be added.
```

Comment:
@andrewfg If you like, you could suggest a description form isResolved as well... The javadoc job is just failing due to the wrong name, and will succeed also without doc for isResolved.

Reply:
Ok.

```
     * @param service the mDNS service to be added.
     * @param isResolved indicates if mDNS has fully resolved the service information. 
```